app/assets/javascripts/admin/templates/api-keys-new.hbs

```diff
@@ -4,24 +4,35 @@
 {{/link-to}}
 
 <div class="api-key">
-  {{#admin-form-row label="admin.api.description"}}
-    {{input value=model.description maxlength="255" placeholder=(i18n "admin.api.description_placeholder")}}
-  {{/admin-form-row}}
+  {{#if model.id}}
+    {{#admin-form-row label="admin.api.key"}}
+      <div>{{model.key}}</div>
```

Comment:
```suggestion
      <span>{{model.key}}</span>
```

Reply:
Using a div here fixes the "double click & copy includes whitespace in firefox" issue. Unfortunately a span does not seem to help 😢

Reply:
Oh ok, didn't know this, totally fine :+1: